.github/workflows/humble-check-docs.yml

```diff
@@ -10,6 +10,10 @@ on:
       - '**.md'
       - '**.yaml'
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
Here don't we need the condition of checking whether it is humble branch or not?

Reply:
we don't check the docs on pushes to the branches, only in PRs. that's why I set this argument to `true`

Reply:
they are cancelled! the other setups are the following: they are only cancelled in PRs, but not if it is pushed to the respective branches

Reply:
Got it